engine-precompiles/src/native.rs

```diff
@@ -400,9 +400,11 @@ impl<I: IO> Precompile for ExitToNear<I> {
             data: exit_event_log.data,
         };
 
+        let gas_cost = Self::required_gas(input)?;
         Ok(PrecompileOutput {
             logs: vec![promise_log, exit_event_log],
-            ..Default::default()
+            cost: gas_cost,
+            output: Default::default(),
```

Comment:
It's better to use a real type of field rather than `Default` here. Also, the clippy suggests doing so.

Reply:
Used `Vec::new()` , is that better?

Reply:
Yep. Or `Vec::default()` absolutely fine. The point is that using a particular type helps you to understand which type is used when you read the code.